src/components/buttons/ha-call-api-button.js

```diff
@@ -1,71 +1,67 @@
-import { html } from '@polymer/polymer/lib/utils/html-tag.js';
-import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { LitElement, html } from "@polymer/lit-element";
 
+import "./ha-progress-button.js";
+import fireEvent from "../../common/dom/fire_event.js";
 
-import './ha-progress-button.js';
-import EventsMixin from '../../mixins/events-mixin.js';
-
-/*
- * @appliesMixin EventsMixin
- */
-class HaCallApiButton extends EventsMixin(PolymerElement) {
-  static get template() {
+class HaCallApiButton extends LitElement {
+  render() {
+    // Arrow function can be removed once this is merged:
+    // https://github.com/Polymer/lit-element/pull/244
     return html`
-    <ha-progress-button id="progress" progress="[[progress]]" on-click="buttonTapped" disabled="[[disabled]]"><slot></slot></ha-progress-button>
-`;
+      <ha-progress-button
+        .progress="${this.progress}"
+        @click="${() => this.buttonTapped()}"
+        ?disabled="${this.disabled}"
```

Comment:
Toggle a boolean based attribute (in JS: el.toggleAttribute('disabled', this.disabled)`)